library-model-loader/src/main/java/edu/ucr/cs/riple/librarymodel/LibraryModelLoader.java

```diff
@@ -120,4 +130,29 @@ public ImmutableSet<MethodRef> nonNullReturns() {
   public ImmutableSetMultimap<MethodRef, Integer> castToNonNullMethods() {
     return ImmutableSetMultimap.of();
   }
+
+  @Override
+  public ImmutableSet<FieldRef> nullableFields() {
+    return nullableFields;
+  }
+
+  @Override
+  public ImmutableList<StreamTypeRecord> customStreamNullabilitySpecs() {
+    return LibraryModels.super.customStreamNullabilitySpecs();
+  }
+
+  /**
+   * Factory interface for creating an instance of type T from a string array.
+   *
+   * @param <T> Type of the instance to create.
+   */
+  interface Factory<T> {
```

Comment:
As noted above, we could eventually have greater type safety here if we could upper bound `T` (not for this PR)

Reply:
Thanks, created the corresponding issue #222.